source/includes/_v2_endpoints.md

```diff
@@ -1441,3 +1441,79 @@ the query.
+
+# Post an image
+
+RestClient.post 'https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post', {:feed_name => "Internal Feed", :type => 'image', :image => 'http://user-topic-images.s3.amazonaws.com/banner/example_image.jpg', :description => 'Example Image', :email => 'admin@backstit.ch'}
+
+# Post a text post
+
+RestClient.post 'https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post', {:feed_name => "Test Source", :type => 'text', :text => "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." , :title => 'Example Text Post', :email => 'admin@backstit.ch'}
+
+```
+
+```shell
+# Post a link
+curl -X POST  https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post?type=link&url=http%3A%2F%2Fwww.wxyz.com%2Fdecodedc%2Fwarning-for-sitting-governors
+
+# Post an image
+curl -X POST  https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post?type=image&image=http%3A%2F%2Fuser-topic-images.s3.amazonaws.com%2Fbanner%2Fexample_image.jpg&description=Example%20image
+
+# Post a text post
+curl -X POST  https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post?type=text&title=Test%20title&text=Lorem+ipsum+dolor+sit+amet%2C+consectetur+adipiscing+elit%2C+sed+do+eiusmod+tempor+incididunt+ut+labore+et+dolore+magna+aliqua.+Ut+enim+ad+minim+veniam%2C+quis+nostrud+exercitation+ullamco+laboris+nisi+ut+aliquip+ex+ea+commodo+consequat.+Duis+aute+irure+dolor+in+reprehenderit+in+voluptate+velit+esse+cillum+dolore+eu+fugiat+nulla+pariatur.+Excepteur+sint+occaecat+cupidatat+non+proident%2C+sunt+in+culpa+qui+officia+deserunt+mollit+anim+id+est+laborum.
+
+```
+
+This endpoint allows you to post a custom result to a custom source. This result will appear in all topics that have the custom source added to them. 
+
+### HTTP Request
+
+`POST https://api.backstit.ch/v2/organizations/{ORGANIZATION_KEY}/post`
+
+### URL Parameters
+
+| Parameter | Required | Description |
+|---------|:-------:|:-----------|
+| ORGANIZATION_KEY | yes | Your organization’s api key is obtained from the organization dashboard under settings. |
+
+### Query Parameters
+
+These parameters are required regardless of the type of result that is being posted.
+
+| Parameter | Required | Description |
+|---------|:-------:|:-------:|:-----------|
+| feed_name | yes | The name of the custom feed that this result will be posted to. |
+| type | yes | The type of result that is being posted to the feed (text, image, or link). |
```

Comment:
I would but something the the description that different parameters are required for each type of post.